### DISCUSSION TITLE
CSK_Module_MultiImageFilter

### DISCUSSION THREAD

**Discussion by golluroSICKAG** [SICKAppSpaceCodingStarterKit/.github#6] [Show and tell]
FYI:
I started to develop a CSK_Module_MultiImageFilter.
See https://github.com/SICKAppSpaceCodingStarterKit/CSK_Module_MultiImageFilter and it fork(s)...
Do not hesitate to contact me if you have some ideas for this module.

**Comment by hauerjuSICKAG** [comment]
I would like to contribute in developing here.  I just started to add more filter types and detailed options to set and made a fork for it.

**Reply by golluroSICKAG** [reply]
Hi!
That sounds awesome! Thx!
Is it already possible to share your ideas what features you are planning to add?
BR

**Comment by golluroSICKAG** [comment]
Pushed version [1.0.0](https://github.com/SICKAppSpaceCodingStarterKit/CSK_Module_MultiImageFilter) today.
